src/QA/QA.Web/QA.Web.csproj

```diff
@@ -14,6 +14,8 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.8" />
     <PackageReference Include="NLog.Web.AspNetCore" Version="5.1.2" />
     <PackageReference Include="SFA.DAS.Configuration.AzureTableStorage" Version="3.0.84" />
+    <PackageReference Include="SFA.DAS.GovUK.Auth" Version="0.2.203" />
```

Comment:
shouldnt need this package?

Reply:
not sure about this, it came as part of Master branch merge? shall i remove it?

Reply:
i think, this came as part of branch updating with the master, let me check it and get came to you, if the references are used any where inside the project.